<?xml version="1.0"?>
<launch>
    <arg name="map_filename" default="my_map"/>

    <node pkg="map_server" type="map_saver" name="$(anon map_saver)"
        args="-f $(find snap_misc)/maps/$(arg map_filename)"/>

</launch>
